wolfcrypt/src/hmac.c

```diff
@@ -1215,11 +1215,18 @@ int wolfSSL_GetHmacMaxSize(void)
         word32 outIdx = 0;
         word32 hashSz = wc_HmacSizeByType(type);
         byte   n = 0x1;
+        word32 N = 0;  /* rf5869: N = ceil(L/HashLen)*/
```

Comment:
Why add an additional variable to the stack when we can just check that L <= (255 * hashSz)? Also please put a comment on why 255 is used here to explain the hard set number

Reply:
@JacobBarthelmeh, Good point.  Fixed.  FYI. The additional variable was for clarity and I wanted to avoid multiplications.